app/gui/view/src/root.rs

```diff
@@ -0,0 +1,181 @@
+//! Root View of the IDE.
+//!
+//! The main entry point to the IDE which can display either Welcome Screen or Project View.
+//! Initially displays Welcome Screen. Lazily initializes Project View on `switch_view_to_project`
+//! call.
+
+use ensogl::prelude::*;
+
+use enso_frp as frp;
+use ensogl::application;
+use ensogl::application::Application;
+use ensogl::display;
+use std::rc::Rc;
+
+
+
+// =============
+// === Model ===
+// =============
+
+/// Two possible states of Root View.
+#[derive(Clone, Debug, PartialEq)]
+enum State {
+    /// Displaying Welcome Screen.
+    WelcomeScreen,
+    /// Displaying Project View with some opened project.
+    OpenedProject,
+}
+
+/// Root View model. Stores both Welcome Screen and Project views and handles their
+/// visibility.
+#[derive(Clone, CloneRef, Debug)]
+pub struct Model {
+    app:            Application,
+    logger:         Logger,
+    display_object: display::object::Instance,
+    state:          Rc<CloneCell<State>>,
+    status_bar:     crate::status_bar::View,
+    welcome_view:   crate::welcome_screen::View,
+    project_view:   Rc<CloneCell<Option<crate::project::View>>>,
+}
+
+impl Model {
+    /// Constuctor. Creates both Welcome Screen and Project View.
```

Comment:
Update the docs. We don't create Project View anymore.

Reply:
Fixed, thank you